src/cdk/table/table.ts

```diff
@@ -169,18 +169,18 @@ export class CdkTable<T> implements CollectionViewer {
     }
   }
 
-  ngOnInit() {
+  ngOnInit(): void {
     // TODO(andrewseguin): Setup a listener for scrolling, emit the calculated view to viewChange
     this._dataDiffer = this._differs.find([]).create(this._trackByFn);
   }
 
-  ngAfterContentInit() {
+  ngAfterContentInit(): void {
     this._cacheColumnDefsByName();
     this._columnDefs.changes.subscribe(() => this._cacheColumnDefsByName());
     this._renderHeaderRow();
   }
 
-  ngAfterContentChecked() {
+  ngAfterContentChecked(): void {
```

Comment:
Lifecycle hooks are not necessary. They are being excluded by our dgeni processors.

Reply:
I'm trying to create consistency on typing. If everything else has it, we might as well make these have it too.